dso-l1/src/main/java/com/tc/object/DistributedObjectClient.java

```diff
@@ -697,7 +709,7 @@ private void shutdown(boolean fromShutdownHook, boolean forceImmediate) {
         clientStopped.notifyAll();
       }
       if (this.channel != null && !this.channel.getProductId().isInternal()) {
-        DSO_LOGGER.info("closing down Terracotta Connection hook=" + fromShutdownHook + " force=" + forceImmediate + " channel=" + this.channel.getChannelID() + " client=" + this.channel.getClientID());
+        DSO_LOGGER.debug("closing down Terracotta Connection hook=" + fromShutdownHook + " force=" + forceImmediate + " channel=" + this.channel.getChannelID() + " client=" + this.channel.getClientID());
       }
```

Comment:
I believe this to be important information worthy of INFO level logging.  What is the rationale behind this change?